magento/pwa-studio
# Updating the node version in venia-concept

## Description

Allowing the usage of node >= 10.x.

Since pwa-buildpack, upward-js and Webpack that uses node support this version it is safe to use this version in the project.

## Related Issue





Closes #1934.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. (Using nvm) `nvm use 12`.
2. Run `yarn watch:all`.
3. It now should accept the usage of this version.

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
